examples/testapp/tsconfig.tsbuildinfo

```diff
@@ -0,0 +1 @@
ebb436169f72ba7124008e2e64c","impliedFormat":1},{"version":"782dcf1557b3e6519d1db78191d230ff2c7549c1da71b2294f69a3cb4cca2d0d","impliedFormat":1},{"version":"be24cbfdefe7b692b2c84f3ff610aa8c0517c9fd7d100b8f6d98dd7b4be550e7","impliedFormat":1},{"version":"b0ec3ce6a36bcb40a1554e11d601038a42d0d8cd16c29a67003dcb2410e56b42","impliedFormat":1},{"version":"b958f13ff3ebdaff7103a2ff0c24d7ada2f49cfe4b723eaa89fba65b03f3bec7","impliedFormat":1},{"version":"f4d75b7fbb68380679fde87033bfb619ac078d79eab34189629f6cf5c6127df1","impliedFormat":1},{"version":"e931a4020c75235e344f4972d6e23a73ffee549d2a0576d1f7a5757c82030501","impliedFormat":1},{"version":"4f65436a20185d7ff4028326855485b655770e53c493ed7af411a3785bd653b5","impliedFormat":1},{"version":"678683b00d859835e7d7ab81ccfcc788d57dd2b29e1ada15d44d61cb808e9fea","impliedFormat":1},{"version":"651dc6a334299341cc3057407b1ed7fcdcc91f0339f1713c09aaba2d6da2bee4","impliedFormat":1},{"version":"d014db3562f555973d15fecab58825b8646d8a01b0fd6501b85f56d9f487c923","impliedFormat":1},{"version":"a997210bde4019c1f2e9683fe80de51c294d4430f133f0533166f6153d3464ef","impliedFormat":1},{"version":"6087ef19af803274ef202003211def5b2efc5e549255f8ce2bfef2d5848d8a7a","impliedFormat":1},{"version":"6bdb08b565d2b32890adebb2a8dce900c76173556a3654cc3809cf58925d9aed","impliedFormat":1},{"version":"4dd3fcb66b533fecae1e77024724bbf7a884ba368317e6948e893d3719cccd24","impliedFormat":1},{"version":"f8af6ad54cde2942850a3776a4de53a3f5aa6f1d7a3f60bddbe4a0d5ed601740","impliedFormat":1},{"version":"2aa9638699da6a9bbe555d9e905c1b68260721f53ea18f9716ef9db70d1d9a41","impliedFormat":1},{"version":"3140dd143e95f00ff9f4dba890d68f0cf3c74f005763a00e06a8cbb04742e40c","impliedFormat":1},{"version":"dc6b797a3048dba55254f81e9f02613b66b04e74b8178bee17af1becc998d829","impliedFormat":1},{"version":"7e34c4423ed565f1b632ce6f9b81c434f01a854ff6d83fc5fc84acd3be3b0c44","impliedFormat":1},{"version":"92af012d09243cd26663127fd8f1bb6885932c25d5688dd2160fb7908655e2a3","impliedFormat":1},{"version":"c8a7e125a854f660b81fcc53620aa2204df7d440143860fb802d5be57ba1a728","impliedFormat":1},{"version":"3dae24ce4e4d7ee7879d887ae979a92835251bf705e0b4d34097840280212986","impliedFormat":1},{"version":"4e6ae7e9acd9dc219ecca348a61f09664f716faaa12c49059279e149dbd95848","impliedFormat":1},{"version":"79524f7535b8200f8fdefffd0975382b7b72464ab7b254f445067d23eec40870","impliedFormat":1},{"version":"3801e04302bcc3c13d3bbe755a45b450ff795d7ab17cdc0c6df36cf18e62f484","impliedFormat":1},{"version":"d0a751957b209e9d3c955857c497b1d969812da8d710a3aa74db21b6929472a4","impliedFormat":1},{"version":"2524c9364851036b8262830e3e4f040f4ab957bb278b6f0b77ac2dfa0dd8e74e","impliedFormat":1},{"version":"d70747c4aeafeda0d47a7d19cc0849f7ef9d8dfddcb3e8615866cfa9efcdaf53","impliedFormat":1},{"version":"b50697a33176d813500728f1d36bd28c6400d28b310a9aa1722a70d404e774ee","impliedFormat":1},{"version":"a1a0c060008e47940075237c9bf71bfc340ac6ec9df109106eca5dcafa248dfd","impliedFormat":1},{"version":"09ee4a7ff5d73eaa62a0e56a58383a
```

Comment:
Should this file be gitignored?

Reply:
[7e8377b](https://github.com/coinbase/coinbase-wallet-sdk/pull/1489/commits/7e8377bea347bb7b11e7f4f065285e4dd7fade16)